packages/emulators/standalone/yabasanshiro-sa/scripts/start_yabasanshiro.sh

```diff
@@ -12,6 +12,7 @@ ROM_DIR="/storage/roms/saturn/yabasanshiro"
 CONFIG_DIR="/storage/.config/yabasanshiro"
 SOURCE_DIR="/usr/config/yabasanshiro"
 BIOS_BACKUP="/storage/roms/bios/yabasanshiro"
+SAVESTATE_DIR="/storage/roms/savestates/saturn/yabasanshiro/"
 
 if [ ! -d "${ROM_DIR}" ]
```

Comment:
```suggestion
mkdir -p "$ROM_DIR" "$BIOS_BACKUP" "$CONFIG_DIR" "$SAVESTATE_DIR"
```
mkdir's -p already omits existing directories.